### PR TITLE
bump requirements to 22.01

### PR DESCRIPTION
**Pull request by bernt-matthias**
and see what breaks

### CODE REVIEW

File: planemo/workflow_lint.py [hunk 1]
```diff
@@ -233,18 +233,16 @@ def _check_test_assertions(lint_context, assertion_definitions):
...
-                    signature = inspect.signature(module.__dict__[function_name])
-                    try:
-                        # try mapping the function with the attributes supplied and check for TypeError
-                        signature.bind("", **assertion_definitions[function_name.split("assert_")[-1]])
-                    except AssertionError:
-                        pass
-                    except TypeError as e:
-                        lint_context.error(f"Invalid assertion in tests: {function_name} {str(e)}")
-                        assertions_valid = False
+        for function_name, function in asserts.assertion_functions.items():
```
Comment (mvdbeek) [inline planemo/workflow_lint.py:236]: I think that means we also need to pin a minimum galaxy-tool-util version (or make this compatible with both)

Comment (bernt-matthias) [reply planemo/workflow_lint.py:236]: I see. So I just require `>=22.01` for all (or is is `>=22.1`?

Comment (mvdbeek) [reply planemo/workflow_lint.py:236]: It's 22.1

File: requirements.txt [hunk 2]
```diff
@@ -3,9 +3,9 @@ bioblend>=0.17.0
 click!=8.0.2
 cwltool>=1.0.20191225192155
 ephemeris>=0.10.3
-galaxy-containers<22.01
-galaxy-tool-util>=21.1.0.dev4,<22.01
-galaxy-util>=20.5.0,<22.01
+galaxy-containers<22.05
+galaxy-tool-util>=21.1.0.dev4,<22.05
```
Comment (mvdbeek) [inline requirements.txt:7]:
```suggestion
galaxy-tool-util>=21.1,<22.5
```

File: requirements.txt [hunk 3]
```diff
@@ -3,9 +3,9 @@ bioblend>=0.17.0
 click!=8.0.2
 cwltool>=1.0.20191225192155
 ephemeris>=0.10.3
-galaxy-containers<22.01
-galaxy-tool-util>=21.1.0.dev4,<22.01
-galaxy-util>=20.5.0,<22.01
+galaxy-containers<22.05
```
Comment (mvdbeek) [inline requirements.txt:6]:
```suggestion
galaxy-containers<22.5
```

File: requirements.txt [hunk 4]
```diff
@@ -3,9 +3,9 @@ bioblend>=0.17.0
...
+galaxy-tool-util>=21.1.0.dev4,<22.05
+galaxy-util>=20.5.0,<22.05
```
Comment (mvdbeek) [inline requirements.txt:8]:
```suggestion
galaxy-util>=20.5.0,<22.5
```

**Comment by bernt-matthias** [comment]
Not sure yet what causes `CmdAutoupdateTestCase.test_autoupdate_workflow` to fail. Any idea?

The other two should be fixed by https://github.com/galaxyproject/galaxy/pull/14496 .. I hope

**Comment by mvdbeek** [comment]
That's https://github.com/galaxyproject/galaxy/pull/14494

**Comment by bernt-matthias** [comment]
> That's [galaxyproject/galaxy#14494](https://github.com/galaxyproject/galaxy/pull/14494)

Cool. But then we need to backport this PR to 22.01 or we wait for 22.05 and bump again.

**Comment by bernt-matthias** [comment]
Not needed anymore .. fixed upstream.

**Comment by mvdbeek** [comment]
No, that is the refactoring action called within the Galaxy instance that planemo spins up.

**Review by nsoranzo — APPROVED** [review]

**Comment by bernt-matthias** [comment]
Should we repeat thin for 22.05?

**Pull request merged**